src/main/java/com/prgms/vouchermanager/contorller/customer/CustomerController.java

```diff
@@ -0,0 +1,40 @@
+package com.prgms.vouchermanager.contorller.customer;
+
+import com.prgms.vouchermanager.domain.customer.Customer;
+import com.prgms.vouchermanager.domain.voucher.Voucher;
+import com.prgms.vouchermanager.service.customer.CustomerService;
+import com.prgms.vouchermanager.util.io.ConsoleInput;
+import org.springframework.stereotype.Controller;
+
+import java.util.List;
+
+import static com.prgms.vouchermanager.contorller.customer.CustomerMenuType.*;
+
+@Controller
+public class CustomerController {
+
+    private final CustomerService customerService;
+    private final ConsoleInput consoleInput;
+
+    public CustomerController(CustomerService customerService, ConsoleInput consoleInput) {
+        this.customerService = customerService;
+        this.consoleInput = consoleInput;
+    }
+
+    public void run() {
+        int menu = 0;
+        try {
+            menu = consoleInput.inputCustomerMenu();
+        } catch (RuntimeException e) {
+            System.out.println(e.getMessage());
+            return;
+        }
+        if (menu == BLACK_LIST.getNumber()) {
+
+            List<Customer> blackList = customerService .getBlackList();
+
+            blackList.stream()
+                    .forEach(customer -> System.out.println(customer.toString()));
```

Comment:
```suggestion
            blackList
                    .forEach(customer -> System.out.println(customer.toString()));
```

list는 forEach api가 별도로 존재합니다. list를 stream() api의 forEach를 사용하고 있는 것 같습니다.

```
    default void forEach(Consumer<? super T> action) {
        Objects.requireNonNull(action);
        for (T t : this) {
            action.accept(t);
        }
    }
```